multifm/fm_demod.c

```diff
@@ -50,29 +51,57 @@ aresult_t multifm_fm_demod_process(struct demod_base *demod, int16_t *in_samples
 
     dfm = BL_CONTAINER_OF(demod, struct multifm_fm_demod, demod);
 
+    // Calculate average power of samples
+    unsigned long sum_smp_pk = 0;
```

Comment:
We should only check the average power of the samples in the given buffer iff CSQ is enabled.